cmd/root.go

```diff
@@ -221,7 +221,7 @@ func searchSSHIdentity() (string, error) {
 	}
```

Comment:
searchSSHIdentity is used to search for a suitable ssh identity file name, not the content. So i think we should reuse this function for the filename in machine.go on both places: machineCreate and machine Console

Reply:
Hi Stefan, you are right, I did some changes